### PR TITLE
Adds Node 5 to allow_failures list

### PR DESCRIPTION
**Pull request by twokul**
@rwjblue @stefanpenner 

screen shot on 2015-11-18 at 12-44-41

### CODE REVIEW

**Comment by rwjblue** [comment]
@homu r+

**Comment by stefanpenner** [comment]
LGTM

File: .travis.yml [hunk 1]
```diff
@@ -18,6 +18,9 @@ install:
   - npm --version
   - git --version
   - npm install --no-optional
+matrix:
+  allow_failures:
+    - node: "5"
```
Comment (trabus) [inline .travis.yml]: Does this need to be `node_js:` instead?

Comment (twokul) [inline .travis.yml]: just updated

**Pull request merged**